app/models/like.rb

```diff
@@ -0,0 +1,8 @@
+class Like < ApplicationRecord
+  belongs_to :user
+  belongs_to :post
+
+  def self.like_counter(post_id)
+    Post.find(post_id).likes.count
+  end
```

Comment:
[OPTIONAL] I think it would be better if you use Rails Active Callbacks like after_save instead of adding self. to the method and make your method private. This will help keep your code DRY and make it more secure